utils/context/user.js

```diff
@@ -46,10 +46,14 @@ const UserProvider = ({ children }) => {
       email: email,
       password: password
     })
-    if (user.firstLogin) {
-      router.push('/create-account')
+
+    if (user) {
+      router.push(
+        `/${user.user_metadata.type === 'Label' ? 'labels' : 'artists'}/${
+          user.user_metadata.slug
+        }`
+      )
```

Comment:
What if you did something like this:

```js
router.push(
  `/${user.user_metadata.type.toLowerCase() + "s"}/${user.user_metadata.slug}`
)
```
But that could be brittle if you had additional types—which you won't, but maaaaybe—so it might be helpful to manage that info in a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) object
```js
const userTypePaths = new Map([
  ['Label', 'labels'],
  ['Artist', 'artists'],
]);
```
and then reference it like this
```js
router.push(
  `/${userTypePaths.get(user.user_metadata.type)}/${user.user_metadata.slug}`
)
```